local.properties

```diff
@@ -1 +1,8 @@
-sdk.dir=C:\\AndroidSDK
\ No newline at end of file
+## This file must *NOT* be checked into Version Control Systems,
```

Comment:
Why is this file checked into Git? The comments suggest it should be removed from the Git repo and listed in a .gitignore file.